server/graphql/v1/mutations/paymentMethods.js

```diff
@@ -195,3 +195,32 @@ export async function updatePaymentMethod(args, remoteUser) {
 
   return models.PaymentMethod.update(pick(args, allowedFields), { where: { id: paymentMethod.id } });
 }
+
+/** Update payment method with given args */
+export async function replaceCreditCard(args, remoteUser) {
+  const oldPaymentMethod = await models.PaymentMethod.findByPk(args.id);
+  if (!oldPaymentMethod || !remoteUser || !remoteUser.isAdmin(oldPaymentMethod.CollectiveId)) {
+    throw PaymentMethodPermissionError;
+  }
+
+  const createArgs = { ...omit(args, ['id']), service: 'stripe', type: 'creditcard' };
```

Comment:
As a general practice I think it's better to whitelist (`pick`) the fields you want rather than blacklist (`omit`). The reason is that any parameter added to the graphql endpoint will otherwise end up here without the implementer knowing it.

Reply:
Updated to use pick instead of omit